examples/test/test_bert_cuda_gpu_aml.py

```diff
@@ -0,0 +1,35 @@
+# -------------------------------------------------------------------------
```

Comment:
Can you add this test to https://github.com/microsoft/Olive/blob/main/examples/test/test_bert_ptq_cpu_aml.py? We have too many bert tests I'm already confused now lol.

Reply:
I separated them since the aml tests take time so it's better to run them in parallel.

Reply:
the bert cpu aml already itself takes over an hour. cuda aml also takes time since it needs to queue for the gpu cluster.

Reply:
I think we need to spend some time in the future to re-evaluate our example tests and probably break them out into their own pipeline.